data/datasets/folk_rnn.py

```diff
@@ -69,13 +69,17 @@ def download(self) -> None:
         if self.data_type == "tokenized_ABC":
             self.url = "https://raw.githubusercontent.com/IraKorshunova/folk-rnn/master/data/data_v2"
 
+            dest_path = os.path.join(
+                self.root,
+                "train",
+            )
+            file_path = os.path.join(dest_path, "data_v2.txt")
+            if not self.replace_if_exists and os.path.exists(file_path):
+                print("Dataset already exists. Skipping download.")
+                return
             response = requests.get(self.url)
```

Comment:
Since we now have a generic Downloader (which is better than what we do here I think), maybe it is a good idea to use it here. Or we can just ignore it and maybe leave a TODO - this is **super low-priority**

Reply:
Might do later